parts/k8s/kubernetescustomscript.sh

```diff
@@ -546,6 +566,14 @@ if [[ "$CONTAINER_RUNTIME" == "clear-containers" ]]; then
 		installClearContainersRuntime
 	fi
 fi
+
+if [[ "$CONTAINER_RUNTIME" == "kata-containers" ]]; then
+	# Ensure we can nest virtualization
+	if grep -q vmx /proc/cpuinfo; then
```

Comment:
what happens if this is false?

Reply:
in that case, the given node would not install kata container artifacts

Reply:
It'll be fully functional, though the user may hope to use Kata, but in reality being using `runc`. 
When Kata is installed, the operator deploying workloads would have option of either using runc or kata-runtime.  In the case VMX isn't supported on the node, any workloads targeting `kata-runtime` would be handled by the default runtime, `runc`.

Reply:
Sounds good, thanks for clarifying